shared/devices/dumb.js

```diff
@@ -1,75 +1,93 @@
 // @flow
-import Devices from './render'
-import type {Device} from '../constants/types/more'
+import Devices from './container'
```

Comment:
This sheet needs to drive things w/ a mock store as it holds the rows which are connected themselves